### PR TITLE
adding casefile for DIVD case DIVD-2024-00039

### PR DESCRIPTION
**Pull request by WesselDIVD**
Casefile addition including all information on DIVD case DIVD-2024-00039

### CODE REVIEW

File: _cases/2024/DIVD-2024-00039.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: case
+title: "Incorrect Authorization vulnerability in Apache OFBiz resulting in RCE"
+author: Wessel Baltus
+lead: Wessel Baltus
+excerpt: "In Apache OFBiz, version 18.12.14and below, a Incorrect Authorization vulnerability exists that allows pre-authentication remote code execution (RCE) resulting in a attacker being able to execute arbitrary commands on the affected system by sending a specially crafted HTTP request."
```
Comment (JstRelax) [inline _cases/2024/DIVD-2024-00039.md:6]:
```suggestion
excerpt: "In Apache OFBiz, version 18.12.14 and below, an Incorrect Authorization vulnerability exists that allows pre-authentication remote code execution (RCE) resulting in an attacker being able to execute arbitrary commands on the affected system by sending a specially crafted HTTP request."
```

File: _cases/2024/DIVD-2024-00039.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+---
+
+## Summary
+
+CVE-2024-38856 is a critical pre-authentication remote code execution (RCE) vulnerability in Apache OFBiz. The flaw stems from insufficient validation of the ProgramExport endpoint, which can be accessed without authentication. Attackers exploit this by chaining the ProgramExport endpoint with other publicly accessible endpoints, effectively bypassing authentication controls. This allows the execution of arbitrary code on vulnerable systems, leading to full system compromise. The vulnerability affects versions of OFBiz up to 18.12.14, and upgrading to version 18.12.15 is required to mitigate this threat.
+
+## Recommendations
+
+The Apache OFBiz versions 18.12.14 and below are vulnerable. Upgrade to version 18.12.15 as soon as possible.
```
Comment (JstRelax) [inline _cases/2024/DIVD-2024-00039.md:48]:
```suggestion
The Apache OFBiz versions 18.12.14 and below are vulnerable. Upgrade to version 18.12.15 or higher as soon as possible.
```

File: _cases/2024/DIVD-2024-00039.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+layout: case
+title: "Incorrect Authorization vulnerability in Apache OFBiz resulting in RCE"
```
Comment (JstRelax) [inline _cases/2024/DIVD-2024-00039.md:3]:
```suggestion
title: "Incorrect Authorization vulnerability in Apache OFBiz resulting in Remote Code Execution"
```

**Review by JstRelax — CHANGES_REQUESTED** [review]
Paar wijzigingen, voor de rest prima!

**Review by vcartman — APPROVED** [review]

**Review by vcartman — APPROVED** [review]

**Review by JstRelax — APPROVED** [review]
LGTM

**Pull request merged**